.travis.yml

```diff
@@ -5,9 +5,11 @@ python:
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -qq libdb-dev
+  - mkdir wheelhouse
+  - wget 'http://www.well.ox.ac.uk/~jk/wheelhouse/pysam-0.8.2.1-cp27-none-linux_x86_64.whl' -O 'wheelhouse/pysam-0.8.2.1-cp27-none-linux_x86_64.whl'
```

Comment:
How about just making a version-agnostic symbolic link at your server that points to the version you want and then the name will never change.  If you want you can have a script that makes it version-agnostic.